.github/workflows/test.yml

```diff
@@ -129,13 +129,23 @@ jobs:
         vagrant up
         vagrant ssh-config >> ~/.ssh/config
     - name: "Run tests"
-      # TODO: enable -test.kill-daemon, after Fedora updates containerd to a recent version (Mar 2021)
-      run: ssh default -- "sudo /vagrant/nerdctl.test -test.v"
+      uses: nick-invision/retry@v2
+      with:
+        timeout_minutes: 10
+        retry_on: timeout
```

Comment:
retry_on: error, perhaps

Reply:
you're right, it can keep the CI for the flaky test. Modification done